README.md

```diff
@@ -311,3 +312,38 @@ kept the flags field for debug. In hind sight, it would have made more sense to
 use a separate header such as `X-B3-Debug: 1` as no other flags were added and
 no implementation treats `X-B3-Flags` as a bit field. That said, renaming the
 field would cause more damage than good, so it was left alone.
+
+## What is the relationship between B3 context and trace data formats?
+
+When you see B3 headers of any kind, you cannot assume the out-of-band data is
+in Zipkin format, or even that data is sent to a tracing system. For example,
+some embed trace data into log lines correlated with the same IDs in the headers.
+Others use B3 headers for dramatically different trace formats.
+
+That said, it is typically the case that data is sent out-of-band, and outgoing
+fields match some fields in that data. For example, in [Zipkin format](https://github.com/openzipkin/zipkin-api), all fields
+except Accept and Deny flags are reported. That's because Accept is implied and
+Deny would not result in reported data.
+
+### Handling B3 when reporting data to a tracing system
+
+It is possible that a proxy, such as Envoy, accepts trace data correlated with B3
+headers sent by the application. Depending on configuration, that proxy could
+start a trace for the act of sending trace data, amplifying traffic to your
+tracing system with low or negative value traces. While most sites will not trace
+data sent to a tracing system, there are generally 2 ways to address this proxy
+setup:
+
+1. Configure the proxy to not trace your trace reporting endpont (Ex in Zipkin, `POST /api/v2/spans`)
```

Comment:
Aside to anyone who can control the code of proxies:

It would be a much better solution for them to add the endpoint they have configured for their tracer to a "deny list" for HTTP sampling policy. In other words, the sampling policy dictated by users would collaborate with the system understood concern like this. Users don't need to do any config maintenance this way.

Such an approach would be safe as intentionally tracing the tracing system is edge case, and tracing the tracing system through a proxy, is even more so.

Understanding some changes to proxies aren't accepted or can take many months to merge, then even longer to roll out, this is more mentioned to help the long term eventually become fixed.